bin/fin

```diff
@@ -4545,6 +4546,20 @@ run_cli ()
 	fi
 }
 
+# Reload Cron Configuration
+cron_reload ()
+{
+	echo-warning "Reloading Cron..."
+	# If crontab file is found within project add contents to user crontab file.
+	if [[ -f "$(get_project_path_dc)/.docksal/services/cli/crontab" ]]; then
```

Comment:
Let us define (somewhere near DOCKSAL_COMMANDS_PATH)

```bash
DOCKSAL_SERVICES_PATH=".docksal/services"
DOCKSAL_CRONTAB_PATH="$DOCKSAL_SERVICES_PATH/cli/crontab"
```

and use `DOCKSAL_CRONTAB_PATH` here. (`DOCKSAL_SERVICES_PATH` is something that will be needed later)